src/utils/intl.ts

```diff
@@ -51,6 +50,26 @@ const isMessageDescriptor = (message?: Message): message is MessageDescriptor =>
 
 const { formatMessage: formatIntlMessage } = intl<ReactNode>();
 
+const addKeyToFormattedMessage = (
```

Comment:
Required when complex message values are included in a component that is part of a list (e.g. actions list item)